lib/logflare/cache_buster.ex

```diff
@@ -21,6 +21,7 @@ defmodule Logflare.CacheBuster do
 
   def handle_info(%Transaction{changes: changes}, state) do
     for record <- changes,
+        :ok = :telemetry.execute([:logflare, :context_cache, :handle_record], %{count: 1}),
```

Comment:
We could add metadata there about type of evicted record.

Reply:
Expensiveness of both approaches is disputable. Many calls should not be super expensive there, but it depends on the handlers attached to Telemetry. `%{count: length(changes)}` needs to traverse whole list again, and additionally do not allows us to group by record type.

Reply:
agreed, having it as is would make sense if we add in the record type.

Reply:
Added!